src/nidcpower/examples/nidcpower_advanced_sequence.py

```diff
@@ -33,28 +33,33 @@ def example(resource_name, channels, options, voltage_max, current_max, points_p
 
         with session.initiate():
             session.wait_for_event(nidcpower.Event.SEQUENCE_ENGINE_DONE)
-            measurements = session.fetch_multiple(points_per_output_function * 2, timeout=timeout)
+            channels = resource_name.split(',')
+            measurement_group = [None] * len(channels)
+            for i, channel in enumerate(channels):
+                channel_name = channel.strip()
+                measurement_group[i] = session.channels[channel_name].fetch_multiple(points_per_output_function * 2, timeout=timeout)
```

Comment:
Can `fetch_multiple` be called for both channels by omitting the repeated capability?

Reply:
No, `fetch_multiple` cannot be called for both channels:

```
Exception has occurred: DriverError
-1074118522: The requested function only allows a single channel to be specified.
```